src/universal/components/ReflectionGroup/ReflectionGroupTitleEditor.js

```diff
@@ -164,7 +165,7 @@ class ReflectionGroupTitleEditor extends Component<Props> {
             <NameInput
               onBlur={this.onSubmit}
               onChange={this.onChange}
-              placeholder="Theme"
+              placeholder={RETRO_TOPIC_LABEL}
```

Comment:
what's the difference between a theme & a topic? could we remove one to reduce the vocab?

Reply:
I basically boiled semantics down to Group and (Upvoted) Topic for simplicity and what I assume to be a less confusing baseline. I sprinkled in more constants to enable easier global changes.